lib/web_console/whiny_object.rb

```diff
@@ -0,0 +1,23 @@
+module WebConsole
+  class WhinyObject < SimpleDelegator
+    def initialize(object, env)
+      @env = env
+      super(object)
+    end
+
+    private
+
+      attr_accessor :env
+
+      def whine_unless(condition)
```

Comment:
@gsamokovarov Actually, I'm a bit confused :confused: 

`whine_unless` is the method name that is currently on [master](https://github.com/rails/web-console/blob/master/lib/web_console/whiny_request.rb#L23)

`log_unless` was the method name I used when the logic was in the middleware, so it's actually was introduced by me.

If understand well your comment about avoiding cosmetic changes, we should use `whine_unless` (so, for example, we wouldn't have to change this line https://github.com/rails/web-console/blob/master/lib/web_console/whiny_request.rb#L8)